package.json

```diff
@@ -54,12 +54,23 @@
     "watch:labextension": "jupyter labextension watch ."
   },
   "dependencies": {
-    "@jupyterlab/application": "^3.1.0"
+    "@deathbeds/jupyterlab-rjsf": "^1.1.0",
+    "@hello-pangea/color-picker": "^3.2.2",
+    "@jupyterlab/application": "^4.0.0-alpha.16",
+    "@jupyterlab/apputils": "^4.0.0-alpha.16",
+    "@jupyterlab/coreutils": "^5.5.1",
+    "@jupyterlab/ui-components": "^4.0.0-alpha.16",
```

Comment:
Could you be consistent for the dependencies?

In particular coreutils 5.5.1 is for JupyterLab 3. If you go for JupyterLab 4 (although I would recommend using JupyterLab 3 & 4), the latest version is 6.0.0-alpha.16.

You can tell that you are supporting 3 and 4 with a version constraint like:

`^3.1.0 || ^4.0.0-alpha.16`

Currently you don't know it but you are downloading multiple versions of the JupyterLab packages for this repo. But you are not seeing any conflict because the final JS assets loaded in JupyterLab does not contain their jupyterlab dependencies.

Reply:
I did not managed to build the extension with a mixed constraints : ^3.1.0 || ^4.0.0-alpha.16. So for the moment I kept JupyterLab 4 dependencies only since building the extension works with them at least locally. The CI crashes due to a conflict on the builder version.